packages/homepage/content/articles/codesandbox-ci-embeds/index.md

```diff
@@ -0,0 +1,95 @@
+---
+banner: ./images/oldvsnew.png
+slug: codesandbox-ci-embeds
```

Comment:
Maybe rename this to
```suggestion
slug: announcing-codesandbox-ci-and-all-new-embeds
```
so it's in line with most of the other slugs? 🤔

Reply:
we're using the url in copy elsewhere, so I don't want to change the slug at this point.

Reply:
OK no problem 🙂